scripts/utils/TitleScreen.gd

```diff
@@ -1,6 +1,6 @@
 extends MarginContainer
 
-func _process(delta):
+func _process(_delta):
```

Comment:
While you're cleaning up, I'd recommend changing this to `_unhandled_input(event)` instead of `_process(_delta)`. No reason to check every frame for keydown if we can just respond to the event when it happens.

Reply:
Did not know of `_unhandled_input`. Great idea. Thanks thanks